dart/lib/src/scope_extensions/_web_scope_extensions.dart

```diff
@@ -0,0 +1,7 @@
+import '../scope.dart';
+
+extension IoScopeExtensions on Scope {
+  Future<void> addAttachementPath(String path) async {
```

Comment:
I think we don't need this method.
Or it could add an attachment from an URL, I'm not sure which makes more sense.

Reply:
no URL afaik, only a path of the local storage, otherwise we'd have to download a file from a URL, we prefer that they do it themselves